build-tools/package.json

```diff
@@ -70,6 +70,7 @@
 		"@commitlint/config-conventional": "^17.6.5",
 		"@commitlint/cz-commitlint": "^17.5.0",
 		"@fluidframework/build-common": "^2.0.0",
+		"@fluidframework/build-tools": "^0.20.0",
```

Comment:
```suggestion
		"@fluidframework/build-tools": "workspace:~",
```

Reply:
that means that `npm run build:fast` doesn't work on first clone?

Reply:
or maybe we don't want fluid-build to build the build-tools? 😄